### PR TITLE
Feat:(UX): Hit the API

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feat:(UX): Hit the API

In this user experience update;

- Create a new game with the name of your choice by using the API
- Enable users to add new scores through the submit button
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -22,8 +22,8 @@ <h2 class="no-margin">Recent Scores</h2>
 
     <form action="#" method="get" class="add-score d-flex">
```
Comment (rloterh) [inline src/index.html]: - [x] OPTIONAL

Kindly ensure that form submission to API should be a `POST` method

File: src/index.js [hunk 2]
```diff
@@ -3,3 +3,19 @@ import Scores from './leaderboard';
...
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const name = form.elements.name.value;
+  const score = form.elements.score.value;
+  leaderboard.addParticipant(name, score).then((res) => {
+    if (!res.error) {
+      form.reset();
+    }
+  });
+});
```
Comment (rloterh) [inline src/index.js:21]: Data is not being stored and retrieved from the API as intended. Kindly ensure that:
- [x] The submit button should hit the POST API endpoint and post user entries.
- [x] When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Blessed, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by blessedjasonmwanza** [comment]
Hello @rloterh ,

Thank you a lot for this review. The API was working and I just created a different one for a fresh start.

Let me quickly take a look at what might be an issue and I'll surely request for another review post successful debug.

> HappyCoding.

File: src/index.html (hunk 1 above)
Comment (blessedjasonmwanza) [reply src/index.html]: Actioned ✅

File: src/index.js (hunk 2 above)
Comment (blessedjasonmwanza) [reply src/index.js:21]: all resolved ✅ . Thanks a lot 🤓

Comment (rloterh) [reply src/index.js:21]: This issue is well resolved! ✔️

File: src/index.html (hunk 1 above)
Comment (rloterh) [reply src/index.html]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Blessed, 

Your project is complete! Great job for making the necessary changes recommended in the previous review(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by blessedjasonmwanza** [comment]
Thank you for the approval @rloterh 🙇🏽 .

> On to making that merge right now 🏃🏽  🚀 

> HappyCoding 🤓 👨🏽‍💻

**Pull request merged**